modules/dop/providers/projectpipeline/deftype/list.go

```diff
@@ -54,3 +54,20 @@ func (p *ProjectPipelineList) Validate() error {
 
 type ProjectPipelineListResult struct {
 }
+
+type ProjectPipelineRefList struct {
```

Comment:
I suggest add `Ref` -> `UsedRef` is more precise.

Reply:
done